### ISSUE TITLE
[Accessibility team] At 200% resize, Create Project page step buttons (Access, Review) are not fully visible

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#303]
[ADO LINK](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/36822622/)

prio 2
sev 3 - Medium

### Actual: 
At 200% resize, Create Project page step buttons (Access, Review) are not fully visible.

### Expected:
At 200% resize, all step buttons should remain Fully visible, Readable, Reachable without loss of content or functionality.

### User Impact:
Low‑vision users who rely on text resizing cannot clearly see or interact with step navigation controls, making it difficult to understand progress or navigate between steps in the Create Project flow.

### Pre-Requisite:
Change the display resolution to 1280*768.
Settings--Accessibility--Text Size-->200%

### Repro Steps:
Resize at 200% zoom.
Launch the AKS desktop application and login with your v-id.
TAB to "Home" page.
Tab to "Projects" tab and press ENTER. Verify all the elements of "Projects" tab.
TAB to "Create Project" button and press ENTER. "Create a Project" dialog will open. Verify the dialog.
TAB to "AKS Managed Project" button and press ENTER.
"New Project" page will open. 
Observe that At 200% resize, Create Project page step buttons (Access, Review) are not fully visible
